App.tsx

```diff
@@ -5,8 +5,8 @@ import Main from "./storybook";
 
 export default props => {
   let [fontsLoaded] = useFonts({
-    anarock_medium: require("./assets/fonts/anarock_medium.ttf"),
-    anarock_regular: require("./assets/fonts/anarock_regular.ttf"),
+    anarock_medium: require("./node_modules/pebble-shared/native/icons/pebble.ttf"),
+    anarock_regular: require("./node_modules/pebble-shared/native/icons/pebble.ttf"),
```

Comment:
this doesnt make any sense. pebble.ttf is icon font, not text font. we use anarock font for all text in sales app & pulse app